intellij/resources/META-INF/plugin.xml

```diff
@@ -12,7 +12,7 @@
         This is the first alpha version of Saros for IntelliJ. Use at your own risk.
     </change-notes>
 
-    <version>0.1.0</version>
+    <version>15.0.0.DEVEL</version>
```

Comment:
Please leave Saros/I at `0.1.0` for the alpha release. Otherwise, this might create unnecessarily high expectations towards the available functionality/stability/quality for the used.